### PR TITLE
style(core/runloop):  style clean for certificate.lua

### PR DESCRIPTION
**Pull request by chronolaw**
### Checklist

- [ ] The Pull Request has tests
- [ ] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* localize some functions
* change `new_tab(1, 0)` to `new_tab(0, 1)` in `fetch_ca_certificates`

### CODE REVIEW

File: kong/runloop/certificate.lua [hunk 1]
```diff
@@ -302,7 +308,7 @@ end
 
 local function fetch_ca_certificates(ca_ids)
   local cas = new_tab(#ca_ids, 0)
-  local key = new_tab(1, 0)
+  local key = new_tab(0, 1)
```
Comment (chronolaw) [inline kong/runloop/certificate.lua:311]: Here key was used by form `key.id`, I think it is a hash table, so we should use `new_tab(0, 1)`.

**Review by ADD-SP — APPROVED** [review]

File: kong/runloop/certificate.lua (hunk 1 above)
Comment (bungle) [reply kong/runloop/certificate.lua:311]: why even create a table when there is only 1?

Comment (bungle) [reply kong/runloop/certificate.lua:311]: You could have module level:
```
local CA_KEY = {
  id = ""
}
```

and then just use:
```
CA_KEY.id = ca_id
local obj, err = kong.db.ca_certificates:select(CA_KEY)
```

Comment (chronolaw) [reply kong/runloop/certificate.lua:311]: good idea, thank you @bungle

**Review by bungle — APPROVED** [review]

**Pull request merged**